### DISCUSSION TITLE
VGM

### DISCUSSION THREAD

**Discussion by muphridz** [vgmtrans/vgmtrans#318] [Music formats]
Hi guys! I wanted to ask you guys if there is any chance of having a converter like vgmtrans (only from .midi to .vgm or .vgz) cool
would there be any chance of you doing this or would it not be possible

**Comment by TheRealHamtaro126** [comment]
"4 YEARS LATER..."

Rainwarrior on NESDEV had once made a converter called NSF2FTM, that by then, also someone got it forked into a seperate GBS2FTM repository by a different person, This can be be helpful (incluing the permission from such project owners of course!)

If this is true, Then it can be turned as a conceptual converter for VGM people want... Just keep creating, dreaming and look up in the skies!!! (or, as some say, Patience is a Virtue, as Coding takes a long hard time, but is worth the reward!)

EDIT: Typo Fixed!

EDIT 2: A "NSF2VGM" converter Already seems to exist in the wild, sadly! I cannot tell you where,

"It's a Secret to Everybody!" - Old man, from the first Legend of Zelda
